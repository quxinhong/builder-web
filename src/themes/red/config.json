{
  "global": {
    "$--color-text-primary": "#FFFFFF",
    "$--color-text-regular": "#A9B7C6",
    "$--color-text-secondary": "#808080",
    "$--color-text-placeholder": "#767676",
    "$--background-color-base": "#505050",
    "$--color-black": "#FFFFFF",
    "$--color-white": "#505050",
    "$--color-primary": "#FE7586"
  },
  "local": {}
}