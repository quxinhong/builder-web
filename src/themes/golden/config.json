{
  "global": {
    "$--color-primary": "#F7AB05",
    "$--color-text-primary": "#FFFFFF",
    "$--color-text-regular": "#A9B7C6",
    "$--color-white": "#505050",
    "$--color-black": "#FFFFFF",
    "$--color-text-secondary": "#808080",
    "$--color-text-placeholder": "#767676",
    "$--background-color-base": "#505050"
  },
  "local": {}
}